topics/cluster-tutorial.md

```diff
@@ -807,8 +835,8 @@ having as a target the empty node.
 Adding a new replica can be performed in two ways. The obvious one is to
 use valkey-cli again, but with the --cluster-replica option, like this:
 
-    valkey-cli --cluster add-node 127.0.0.1:7006 127.0.0.1:7000 --cluster-replica
-
+    `valkey-cli --cluster add-node 127.0.0.1:7006 127.0.0.1:7000 --cluster-replica
+`
```

Comment:
Have you tested this in a browser? I expect this might look weird since the backtick spans a line.